packages/user/src/services/user-module.ts

```diff
@@ -96,6 +101,18 @@ export default class UserModuleService<
       populate: true,
     })
 
+    sharedContext.messageAggregator?.saveRawMessageData(
+      users.map((user) => ({
```

Comment:
suggestion: maybe we should use `composeMessage` so that we normalize events

Reply:
I added `saveRawMessageData` to take the message input for `composeMessage` and call `eventAggregator.save` per. Oli's comment here: https://github.com/medusajs/medusa/pull/6431#discussion_r1494647141

I'm fine with just changing the `save` method as well